.gitignore

```diff
@@ -23,3 +23,4 @@ plugin/Makefile
 plugin/.merlin
 *.out
 _opam
+src/plibrary.ml
```

Comment:
I excluded this from git because it appears to be a build artifact that gets generated whenever `build.sh` is run.